project/eScavenge/Game.py

```diff
@@ -280,8 +280,10 @@ def __init__(self, maker):
         self.maker = maker
 
     def create_game(self, *args, **kwargs):
-        return self.maker(*args, **kwargs)
-
+        if self.game._Game__started:
```

Comment:
self in this case is GameFactory, which has no game member. This condition should always cause an exception.

Reply:
@holyzala Would I be able to fix that by just using Game.started?  instead of self.game.Game__started.  From what I understand in regards to python, calling Game.stared is the class variable shared among all instances of that class.

Reply:
No, at this point we can not be sure there even is a Game in existence. And to check it would require a CLI instance passed down somehow. This check should be done in CLI I think, but definitely not here in the factory.